gpii/node_modules/transformer/src/js/Transformer.js

```diff
@@ -32,6 +32,23 @@ var fluid = fluid || require("infusion"),
         }
     });
 
+    gpii.transformer.filterSupportedSettings = function (settings, supportedSettings) {
```

Comment:
Comments block - also, possibly this belongs in settingsHandlerUtilities - don't we have something remarkably similar to this there already?